test/e2e/autoscale_test.go

```diff
@@ -443,16 +443,16 @@ func TestGracefulScaledown(t *testing.T) {
 		}))
 	defer test.TearDown(ctx.clients, ctx.names)
 
-	autoscalerConfigMap, err := rawCM(ctx.clients, autoscalerconfig.ConfigName)
+	autoscalerConfigMap, err := RawCM(ctx.clients, autoscalerconfig.ConfigName)
```

Comment:
This and next file no longer need modifications?

Reply:
Yeah. Sorry. Will fix it.